sway_libs/src/fixed_point/ufp/ufp64/ufp64.sw

```diff
@@ -0,0 +1,267 @@
+library ufp64;
+//! A wrapper around u64 type for a library for Sway for mathematical functions operating with signed 32.32-bit fixed point numbers.
+use core::num::*;
+use std::assert::assert;
+use std::math::*;
+use std::revert::revert;
+use std::u128::U128;
+use std::math::Exponentiate;
+
+pub struct UFP64 {
+    value: u64,
+}
+
+impl UFP64 {
+    /// Convinience function to know the denominator
+    pub fn denominator() -> u64 {
+        1 << 32
+    }
+
+    pub fn zero() -> Self {
+        Self { value: 0 }
+    }
+
+    /// Creates UFP64 from u64. Note that ~UFP64::from(1) is 1 / 2^32 and not 1.
+    pub fn from(value: u64) -> Self {
```

Comment:
Is this `from` used or should the `From<T>` from the standard library be used?

Reply:
Fixed https://github.com/FuelLabs/sway-libs/pull/31/commits/a944d94d2c8498141b635066ff1429d8d93868fe